src/sentry/api/endpoints/auth_index.py

```diff
@@ -31,6 +33,8 @@ class AuthIndexEndpoint(Endpoint):
 
     permission_classes = ()
 
+    logger: logging.Logger = logging.getLogger(__name__)
```

Comment:
sorry, initializing the logger here is also weird. it should generally be initialized at the top of the file. you can see how this is done in other cases when logger is used for the most part.

Reply:
ah i see, i found one instance of it being initialized in a class in `RuleProcessor` so i thought it was okay. will move

Reply:
yeah it is a bit of a nebulous convention for sure.